htdocs/js/notebook/notebook_controller.js

```diff
@@ -26,6 +26,10 @@ Notebook.create_controller = function(model)
         };
     }();
 
+    var refresh_search = _.debounce(function() {
+        update_notebook(refresh_buffers()).then(function() { RCloud.UI.find_replace.reset(); })
+    }, 350);
+
```

Comment:
This is forcing a save to GitHub which seems like overkill (and disablesj/falsifies the autosave feature that is the rest of `on_dirty`). 

Is this because find/replace will only look at the notebook model content rather than the current live content of the ace editors?